libs/core/src/components/pds-tooltip/pds-tooltip.scss

```diff
@@ -37,6 +37,7 @@
   box-shadow: var(--box-shadow-default);
   color: var(--color-text-default);
   // TODO: need to use block / none but the tooltip content width and height are needed for calculations
```

Comment:
Do we know what this TODO is for and are we able to clean this up while we are in here?

Reply:
@monicawheeler TBH, I have no idea but will check into it.

Reply:
Maybe @QuintonJason could help :) https://github.com/Kajabi/pine/commit/84f6259be506675038d9ca93ea0a343fcf08c770

Reply:
Already pinged him!

Reply:
@monicawheeler checked in with @QuintonJason and that comment ties into the universal approach of how tooltip, dropdown, popover will work. Quinton will be following up here as well as he begins work on the other mentioned components, so no action needed in this PR.

Reply:
Thanks for the follow up!